packages/components/src/components/checkbox/checkbox.tsx

```diff
@@ -60,27 +60,29 @@ const Checkbox = React.forwardRef<HTMLInputElement, TCheckBoxProps>(
                     'dc-checkbox--disabled': disabled,
                 })}
             >
-                <input
-                    className='dc-checkbox__input'
-                    type='checkbox'
-                    id={id}
-                    ref={ref}
-                    disabled={disabled}
-                    onChange={onInputChange}
-                    defaultChecked={checked}
-                    checked={checked}
-                    {...otherProps}
-                />
-                <span
-                    className={classNames('dc-checkbox__box', {
-                        'dc-checkbox__box--active': checked,
-                        'dc-checkbox__box--disabled': disabled,
-                        'dc-checkbox--grey-disabled': disabled && greyDisabled,
-                    })}
-                    tabIndex={withTabIndex}
-                    onKeyDown={handleKeyDown}
-                >
-                    {!!checked && <Icon icon='IcCheckmark' color='active' />}
+                <span>
```

Comment:
Why did you wrap the current elements under a span. The checkbox is a core component, make sure this change doesn't break in other places that is used.

Reply:
The checkbox input element and icon is a single visual element and therefore grouped them using <span> so that i can apply flexbox styling to set the elements alignment according to figma design. This checkbox component is working fine in other places where its used, and in the future if we need to place the checkbox at the top, center or bottom with respect to the text displayed, we simply have to apply flex styling.
![Screenshot 2023-08-04 at 3 16 15 PM](https://github.com/binary-com/deriv-app/assets/122449658/4bb891d4-b741-42b5-8bfd-75e683f84d85)

Reply:
@shaheer-deriv alright 👍🏼  add some sanity to the checkbox in the QA checklist

Reply:
ok.. sure. 👌🏼